keyboards/viktus/minne/keymaps/via/rules.mk

```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```

Comment:
```suggestion
```

Reply:
Can you explain the reasoning for removing this?